DWPF-703 removes email sending logic, notify package and  its dependencies (#24)
uktrade/django-feedback-govuk: README.md

```

Now the make commands should work as expected.

## Sending automated emails after form submission

> Based on the needs of your Service you may want the package to inform you via email of the feedback received. For example you might want to receive an email if there is new feedback to be reviewed and call the method on a regular schedule (e.g. once a day) through cron, Celery Beat, etc.

Reviewer: ```suggestion
Based on the needs of your project you may want to be notified of the feedback received. For example you might want to receive an email if there is new feedback to be reviewed. To do this you can create a method and call it on a regular schedule (e.g. once a day) through cron, Celery Beat, etc.
```
